_data/faq.yml

```diff
@@ -243,6 +243,12 @@
           For monthly billing, we require automated payments via credit card or ACH transfer. If you wish to use invoices and pay by wire transfer or check instead, you must do annual billing.
         </p>
 
+    - question: Need to customize our Terms of Service?
+      answer: |
+        <p>
+          If you wish to customize the contract terms instead of using our standard <a href="/terms/" title="Gruntwork Terms of Service">Terms of Service</a>, you may download our Terms as a <a href="/Gruntwork-TermsOfService.docx" title="Download Gruntwork Terms of Service">Microsoft Word File</a>, but please note all custom contract terms must sign up for a subscription with at least 20 users.
```

Comment:
I'd reverse this phrasing: 

> We recommend that most customers go through our normal checkout flow and accept our online Terms of Service. Custom Terms of Service are only available for "Enterprise Customers" that will be signing up for a Gruntwork Subscription of at least 20 users. If you are an Enterprise Customer, you can download our Terms of Services as a Microsoft Word Document here.

Reply:
Updated in 2590e427fffe5c4939a6f6d59c30765b4db1cadc.